### ISSUE TITLE
[claude-audit] shared: _build_scheduler 'plateau' branch ignores scheduler_prefix

### ISSUE THREAD

**Issue by alexanderdfree** [alexanderdfree/Fantasy_Football_ML_AWS#908]
- **File**: `src/shared/pipeline.py:538`
- **Severity**: MED
- **Area**: shared
- **First seen**: @2900828 on 2026-06-06
- **What**: The recently added per-scheduler prefix-override mechanism (commit 7e92913 'apply batch LR attention defaults') routes the 'onecycle' and 'cosine_warm_restarts' branches through _scheduler_value(cfg, key, scheduler_prefix), so attention paths can set attn_onecycle_max_lr / attn_cosine_t0 / etc. and have them take effect. The 'plateau' branch was missed: it reads cfg['plateau_factor'] and cfg['plateau_patience'] directly, so any attn_plateau_factor / attn_plateau_patience override is silently discarded and the base-NN values are used for the attention run.
- **Why suspect**: scheduler_prefix='attn_' is passed by both _train_attention_nn and _train_nested_attention_nn (pipeline.py:836,938). The override mechanism is uniform across the other two scheduler types, but plateau was left out — a classic half-applied refactor. No production config uses plateau today (all six positions set scheduler_type to onecycle or cosine_warm_restarts), so this is latent, but the override mechanism is broken-by-construction for the plateau path and will silently produce the wrong scheduler if anyone migrates.
- **Suggested action**: Wrap the plateau branch's factor/patience reads the same way: factor=_scheduler_value(cfg, 'plateau_factor', scheduler_prefix), patience=_scheduler_value(cfg, 'plateau_patience', scheduler_prefix).
- **Evidence**: `    elif sched_type == "plateau":
        return torch.optim.lr_scheduler.ReduceLROnPlateau(
            optimizer,
            mode="min",
            factor=cfg["plateau_factor"],
            patience=cfg["plateau_patience"],
        ), False  # scheduler_per_batch=False`

```json
{"schema":"claude-audit/v1","file":"src/shared/pipeline.py","line":538,"severity":"MED","area":"shared","category":"unfinished_pr","first_seen_sha":"2900828"}

```
